### PR TITLE
Auto generate sfog

### PR DESCRIPTION
**Pull request by calebziamba**
Ran a script over all the files Jon sent containing the lyrics and details for the Songs for Our Generation songbook. I think it would be good to scan/proofread because I already ended up fixing a handful of songs that didn't fit the format exactly so they either had the guitar chords appear as lyrics or some of the verses appeared as a single line, and some had weird `presentation` numbering. I fixed those which I noticed, but I'm certain at least some slipped through the cracks.

### CODE REVIEW

**Comment by brotatotes** [comment]
Is there a way to test this locally?

**Comment by calebziamba** [comment]
Oh, you can change the target url to use this branch. Basically revert my
last commit. You have to also change the branch name in the Songs repo and
make sure you have the test flag set in the SongUtils file

File: resources/metadata/sfog.json [hunk 1]
```diff
@@ -1,564 +1,8687 @@
...
-          "To possess by faith what I could not earn:",
-          "All surpassing gift of righteousness"
-        ],
-        "v3": [
-          "Oh, to know the pow’r of Your risen life,",
-          "And to know You in Your sufferings,",
-          "To become like You in Your death my Lord",
-          "So with You to live and never die."
-        ]
+  "Songs for Our Generation": [
```
Comment (brotatotes) [inline resources/metadata/sfog.json]: Should 'for' be capital?

File: resources/songbooks.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
  ],
  "testing": [
    {
-     "name": "Songs For Our Generation",
+     "name": "Songs for Our Generation",
```
Comment (brotatotes) [inline resources/songbooks.json]: Why lowercase "for" ?

**Review by brotatotes — DISMISSED** [review]
Approved with 1 minor comment.

Also, I think you should add the script you wrote to the Utilities repo! It could be very useful in the future!

**Comment by brandonxia01** [comment]
Oh boy this PR looks juicy, Thanks Caleb!

File: resources/songbooks.json (hunk 2 above)
Comment (calebziamba) [reply resources/songbooks.json]: We can capitalize, they just need to match in both files so it can access the array

Comment (brotatotes) [reply resources/songbooks.json]: Let's capitalize then.

**Review by brotatotes — APPROVED** [review]

**Pull request merged**